cfme/infrastructure/config_management.py

```diff
@@ -520,3 +440,142 @@ def __init__(self, name=None, url=None, ssl=None, credentials=None, key=None):
+class RefreshRelationships(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.check(Quadicon('{} Configuration Manager'.format(self.obj.name), None).checkbox())
+        cfg_btn('Refresh Relationships and Power states', invokes_alert=True)
+
+
+@navigator.register(ConfigManager, 'Remove')
+class Remove(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.check(Quadicon('{} Configuration Manager'.format(self.obj.name), None).checkbox())
+        cfg_btn('Remove selected items from the VMDB', invokes_alert=True)
+
+
+@navigator.register(ConfigManager, 'Details')
+class Details(CFMENavigateStep):
+    prerequisite = NavigateToSibling('All')
+
+    def step(self):
+        sel.click(Quadicon('{} Configuration Manager'.format(self.obj.name), None))
+
+    def am_i_here(self):
+        return summary_title() in ('Configuration Profiles under Red Hat Satellite '
+                                   'Provider "{} Configuration Manager"'.format(self.obj.name),
+                                   'Inventory Groups under Ansible Tower Provider'
+                                   ' "{} Configuration Manager"'.format(self.obj.name))
+
+
+@navigator.register(ConfigManager, 'EditFromDetails')
+class EditFromDetails(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        cfg_btn('Edit this Provider')
+
+
+@navigator.register(ConfigManager, 'RefreshRelationshipsFromDetails')
+class RefreshRelationshipsFromDetails(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        cfg_btn('Refresh Relationships and Power states', invokes_alert=True)
+
+
+# todo: not sure whether this works or not. it seems it wasn't used for a long time
+@navigator.register(ConfigProfile, 'Description')
+class Description(CFMENavigateStep):
+
+    def step(self):
+        navigate_to(self.obj.manager, 'Details')
+        tb.select('List View'),
+        page.list_table_config_profiles.click_cell('Description', self.obj.name)
+
+
+@navigator.register(ConfigManager, 'RemoveFromDetails')
+class RemoveFromDetails(CFMENavigateStep):
+    prerequisite = NavigateToSibling('Details')
+
+    def step(self):
+        cfg_btn('Remove selected items from the VMDB', invokes_alert=True)
+
+
+@navigator.register(ConfigSystem, 'All')
+class SysAll(CFMENavigateStep):
+    prerequisite = NavigateToAttribute('appliance', 'LoggedIn')
+
+    def step(self):
+        from cfme.web_ui.menu import nav
+        nav._nav_to_fn('Configuration', 'Configuration Management')(None)
+
+    def resetter(self):
+        accordion.tree('Configured Systems',
+                       version.pick({version.LOWEST: 'All Red Hat Satellite Configured Systems',
+                                     '5.6': 'All Configured Systems'}))
+
+        tb.select('Grid View')
+
+    def am_i_here(self):
+        return summary_title() == 'All Configured Systems'
```

Comment:
Can we get the controller included in the checks here please, @mshriver has some examples I think